index.html

```diff
@@ -687,31 +877,26 @@ <h2 class="subscribe-block__title title">
             <div class="search-container subscribe-wrapper">
               <!-- Search form -->
 
-              <form
-                class="search-container__search-form mail-form"
-                method="get"
-              >
+              <form class="search-container__search-form mail-form">
                 <input
                   class="mail-input"
```

Comment:
Why do you need a special class name here? Is this input so much different from others?

Reply:
Yes, this input have different width